### DISCUSSION TITLE
svpcom/rtl8812au vs. aircrack-ng/rtl8812au ?

### DISCUSSION THREAD

**Discussion by XBeg9** [svpcom/wfb-ng#281] [Questions] [answered]
svpcom/rtl8812au vs. aircrack-ng/rtl8812au

Could you please help me understand the benefit of using your driver instead of aircrack's? Is there a significant difference?

Thanks

**Comment by svpcom** [comment, marked as answer]
I use my fork of rtl8812au for my own needs (to work with wfb-ng). It based on v5.2.20 and included some fixes and improvements for packet injection that are unavailable in aircrack driver. You can see all my commits in commit log

**Comment by sensei-hacker** [comment]
Presumably you PRed those fixes and improvements back to the source you forked from, so most of them will soon be in the Air Crack copy too?

I spent last night trying to get my RTL8812au working. In my regular laptop. If you have fixes, as opposed to things specific to WFB-ng, it would be great if the wifi driver were fixed for everyone, not just WFB-ng users.

**Reply by svpcom** [reply]
Ordinary users don't need packet injection and power boosting which wfb-ng uses. Without additional active cooling it will destroy your card very fast.

**Reply by XBeg9** [reply]
But aircrack-ng has power boosting, or did you do something different?

**Reply by svpcom** [reply]
Only in 5.2.20. All newer versions have it broken
